src/components/FlawComments/FlawComments.vue

```diff
@@ -46,212 +40,90 @@ const commentLabels = computed(() => {
-  const horizontalPos = selectionStart - newComment.value.slice(0, selectionStart).lastIndexOf('\n') - lastWord.length;
-  const verticalPos = elTextArea!.value.slice(0, selectionStart).split('\n').length ?? 0;
-
-  caretPos.value[0] = `calc(${paddingLeft}px + ${horizontalPos}ch)`;
-  caretPos.value[1] = (textAreaOffset + paddingTop + Math.min(lineHeight * verticalPos, textAreaRect.height)) + 'px';
-};
-
-const handleSuggestionClick = (user: any) => {
-  const lastWord = getLastWord();
-  if (!lastWord) {
-    return;
-  }
-
-  const username = `[~${user.name}]`;
-  const contentBefore = newComment.value.slice(0, newComment.value.lastIndexOf(lastWord));
-  const contentAfter = newComment.value.slice(newComment.value.lastIndexOf(lastWord) + lastWord.length);
-
-  newComment.value = contentBefore + username + contentAfter;
-
-  suggestions.value = [];
-  refTextArea.value?.elTextArea?.focus();
-};
-
-const getLastWord = () => {
-  const elTextArea = refTextArea.value?.elTextArea;
-  const selectionStart = elTextArea?.selectionStart ?? 0;
-
-  return newComment.value?.slice(0, selectionStart).split('\n')?.pop()?.split(' ')?.pop();
-};
-
-const clearSuggestions = (event: FocusEvent | KeyboardEvent | MouseEvent) => {
-  if (event instanceof KeyboardEvent
-    && ['ArrowDown', 'ArrowLeft', 'ArrowRight', 'ArrowUp', 'Enter', 'Escape', 'Tab']
-      .includes(event.key)) {
-    suggestions.value = [];
-    return;
-  }
-
-  if (event instanceof MouseEvent) {
-    suggestions.value = [];
-    return;
-  }
-
-  if (event instanceof FocusEvent
-    && event.relatedTarget !== event.currentTarget
-    && !(event.currentTarget as Node)?.contains(event.relatedTarget as Node)
-  ) {
-    suggestions.value = [];
-  }
-};
+const newCommentAllowed = computed(() =>
+  (
+    (
+      (
+        selectedTab.value !== CommentType.System
+        && (props.internalCommentsAvailable || selectedTab.value !== CommentType.Internal)
+      )
+      || (settings.value['unifiedCommentsView']))
+  ),
+);
+
+const showBugzillaLink = computed(() =>
+  props.commentsByType[CommentType.Private].length > 0
+  || props.commentsByType[CommentType.Public].length > 0
+  || props.commentsByType[CommentType.System].length > 0,
+);
+
+const showJiraLink = computed(() =>
+  props.internalCommentsAvailable && props.commentsByType[CommentType.Internal].length > 0,
+);
 </script>
 
 <template>
-  <section class="osim-comments my-2">
-    <h4 class="mb-4">Comments</h4>
+  <section class="osim-comments">
+    <h4>Comments</h4>
+    <CommentsToolbar
+      :newCommentAllowed
+      :showBugzillaLink
+      :bugzillaLink
+      :showJiraLink
+      :isSaving
+      :taskKey
+      :isAddingNewComment
+      :newCommentType
+      :internalCommentsAvailable
+      @startNewComment="(type) => startNewComment(type)"
+    />
+    <div v-if="!internalCommentsAvailable" class="alert alert-danger">
```

Comment:
as commented in the code review:
- Add a conditional to only show this alert in the unified view
- Disable the tab and update the tooltip/add an icon to indicate that is not available